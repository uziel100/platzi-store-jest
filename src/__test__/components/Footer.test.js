import React from 'react';
import { mount } from 'enzyme';
import Footer from '../../components/Footer';
import {  create } from 'react-test-renderer'

describe('<Footer />', () => {
  const footer = mount(<Footer />);
  test('Render del componente Footer', () => {
    expect(footer.length).toEqual(1);
  });

  test('render un texto', () => {
    expect( footer.find('.Footer-title').text()).toEqual('Platzi Store')  
  })
  
});


describe('footer snapshot', () => {
  test('comprobar la ui del componente', () => {
    const footer = create(<Footer />)
    expect(footer.toJSON()).toMatchSnapshot()
  })
})
