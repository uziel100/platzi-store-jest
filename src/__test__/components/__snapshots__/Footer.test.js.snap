// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer snapshot comprobar la ui del componente 1`] = `
<div
  className="Footer"
>
  <p
    className="Footer-title"
  >
    Platzi Store
  </p>
  <p
    className="Footer-copy"
  >
    Todos los Izquierdos Reservados
  </p>
</div>
`;
